tracing/src/lib.rs

```diff
@@ -817,23 +817,23 @@
 //! long as doing so complies with this policy.
 //!
 //! [`log`]: https://docs.rs/log/0.4.6/log/
-//! [span]: span/index.html
-//! [spans]: span/index.html
-//! [`Span`]: span/struct.Span.html
-//! [`in_scope`]: span/struct.Span.html#method.in_scope
-//! [`Event`]: struct.Event.html
-//! [event]: struct.Event.html
-//! [events]: struct.Event.html
-//! [`Subscriber`]: subscriber/trait.Subscriber.html
-//! [Subscriber::event]: subscriber/trait.Subscriber.html#tymethod.event
-//! [`enter`]: subscriber/trait.Subscriber.html#tymethod.enter
-//! [`exit`]: subscriber/trait.Subscriber.html#tymethod.exit
-//! [`enabled`]: subscriber/trait.Subscriber.html#tymethod.enabled
-//! [metadata]: struct.Metadata.html
-//! [`field::display`]: field/fn.display.html
-//! [`field::debug`]: field/fn.debug.html
-//! [`set_global_default`]: subscriber/fn.set_global_default.html
-//! [`with_default`]: subscriber/fn.with_default.html
+//! [span]: mod@span
+//! [spans]: mod@span
+//! [`Span`]: span::Span
+//! [`in_scope`]: span::Span::in_scope
+//! [`Event`]: Event
```

Comment:
This will be resolved automatically, you don't need the extra reference link :) rustdoc strips the backticks
```suggestion
```

Reply:
Thanks! I've accepted your suggestion!